settings.gradle

```diff
@@ -7,6 +7,7 @@ includeWithFlags ':bom',                       'bom'
 
 // Published Java projects
 includeWithFlags ':core',                       'java', 'publish', 'shade', 'trim'
+includeWithFlags ':docs-client',                'publish'
```

Comment:
Could we avoid publishing this but just use the build output into the resource set of `:core`?

Reply:
Yep both would work, though it would require a bit more configuration. I thought this might be a good idea to separate though since it's then very easy to exclude the jar to exclude the webapp if you don't care about it. What do you think?

Reply:
I'd still prefer making this part of the core JAR unless it adds too much. ;-)